lib/services/data/db/mongodb_provider.dart

```diff
@@ -31,7 +31,7 @@ class MongoDbProvider {
 	}
 
 	Future replaceAll(Collection collection, List<SelectorBuilder> selectors, List<Map<String, dynamic>> documents) {
-		return _execute(() => _client.collection(collection.name).updateAll(selectors, documents, upsert: true));
+		return _execute(() => _client.collection(collection.name).update(selectors, documents, upsert: true));
```

Comment:
Why?

Reply:
I don't know why it wasn't available for me, now its ok

Reply:
Probably because I've just added it :)